dotcom-rendering/src/web/lib/liveblogAdSlots.ts

```diff
@@ -8,12 +8,22 @@ const MAX_INLINE_ADS = 8;
 /**
  * Minimum allowed space between the top of the liveblog container and the highest inline ad.
  */
-const MIN_SPACE_BEFORE_FIRST_AD = 1_000;
+const MIN_SPACE_BEFORE_FIRST_AD = 500;
```

Comment:
Currently an advert will appear after the first block. This means it can be side-by-side with the right ad. Ensuring approximately 500px between the top of the liveblog and the first ad means that this will be unlikely to happen